internal/profiles/service.go

```diff
@@ -276,6 +291,63 @@ func (p *profileService) UpdateProfile(
 	return profile, nil
 }
 
+func (p *profileService) PatchProfile(
+	ctx context.Context,
+	projectID uuid.UUID,
+	profileID uuid.UUID,
+	subscriptionID uuid.UUID,
+	patch *minderv1.Profile,
+	updateMask *fieldmaskpb.FieldMask,
+	qtx db.Querier,
+) (*minderv1.Profile, error) {
+	oldProfilePb, err := getProfilePBFromDB(ctx, profileID, projectID, qtx)
```

Comment:
Do you want to call `old` "base" here?  It's slightly confusing, because we go through the steps and then pass `old` to the Update method.

Reply:
thanks, renamed